source/administration/config-settings.rst

```diff
@@ -1608,6 +1608,9 @@ Enable cross-origin requests from
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Enable HTTP cross-origin requests from specific domains separated by spaces. Type ``*`` to allow CORS from any domain or leave it blank to disable it.
 
+.. note::
+  If the domain entered and the URL are you using to access the site to make the configration changes in the System Console do not match, you will lose access to the System Console.  Please make sure you have entered this information correctly before saving.  If you do experience lost access System Console after changing this setting, you can remove or reset your entry in the config.json to restorce access. 
```

Comment:
This isn't accurate. The URLs you enter here __do not__ have to match your SiteURL. It's just if you put anything in this field other than `*` or leaving it blank, you __must__ have your SiteURL setting configured. To be clear: you don't need to put your site URL in here ever.

Reply:
Thank you for clarifying @jwilander.  I revised to reflect what you have described here.